collect

```diff
@@ -5,6 +5,7 @@ Bundler.setup(:default)
 
 $LOAD_PATH << "./lib"
 
+require "exception_handler"
 require "govuk_sli_collector"
 
-GovukSliCollector.call
+ExceptionHandler.report { GovukSliCollector.call }
```

Comment:
What kind of errors would not be swallowed i.e., raised outside of PublishingLatencySli?

Reply:
You're right. Currently, without any code changes, there's nothing that can break that wouldn't already be broken.

Having this was useful while I was making recent changes, so I figured it makes sense to leave it to cover future changes, too. Does it feel too "future use case" to you?